File: internal/sem/testdata/langmatrix/python/ledger.py
class Ledger:
    def __init__(self) -> None:
        self.total = 0

    def add(self, amount: int) -> int:
        return self.total + amount


def ledger_double(amount: int) -> int:
    ledger = Ledger()
    return ledger.add(amount) * 2

File: internal/sem/testdata/langmatrix/cpp/ledger.cpp
#include <string>

class Ledger {
public:
    int Add(int amount) const;

private:
    int total_ = 0;
};

int Ledger::Add(int amount) const {
    return total_ + amount;
}

int LedgerDouble(int amount) {
    Ledger ledger;
    return ledger.Add(amount) * 2;
}
